interface/super/edit_list.php

```diff
@@ -1238,11 +1311,25 @@ function mysubmit() {
              *   If the edit_options setting of the main list entry is set to 1,
              *    then the list items with edit_options set to 1 will show.
              */
+
+            /**
+             * In case when we are have a get parametr "list_id_container", we are set manually variable list_id
+             * and range from...to if these is set
+             * adding 1 to from and to limits
+             */
+//            $list_from = ($list_from <= 0 ? 0 : $list_from);
```

Comment:
better to remove code than comment it out

Reply:
Now is clean.